### PR TITLE
ISSUE-687 More JMAP filters action

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Review by Arsnael — COMMENTED** [review]
Ah rule action on multiple mailboxes comes for free too as an extra? :)

Can you then though change the line in the doc regarding `mailboxIds` and remove the last sentence : `Must be a single value.` ? 

For the rest, looks good to me :)

**Comment by chibenwa** [comment]
> Ah rule action on multiple mailboxes comes for free too as an extra? :)

That's thanks to storage directive in several folders

> Can you then though change the line in the doc regarding mailboxIds and remove the last sentence : Must be a single value. ?

:+1:

**Comment by quantranhong1999** [comment]
> Ah rule action on multiple mailboxes comes for free too as an extra? :)

@QuangHoang1210 you would have your wished feature soon (filtering targets multiple mailboxes) :-)

File: docs/modules/ROOT/pages/tmail-backend/jmap-extensions/jmapFilters.adoc [hunk 1]
```diff
@@ -40,11 +40,15 @@ The comparator is case-incentive for `from`, `to`, `cc`, `recipient` fields but
 
 An `Action` object has the following properties:
 
+- **markAsSeen** `Boolean` optional defaults to false. Should the matching email be marked as seen.
+- **markAsImportant** `Boolean` optional defaults to false. Should the matching email be marked as important.
+- **reject** `Boolean` optional defaults to false. Should the matching email be rejected.
+- **withKeywords** `String[]` optional defaults to empty. The keyword to apply to matching emails.
```
Comment (quantranhong1999) [inline docs/modules/ROOT/pages/tmail-backend/jmap-extensions/jmapFilters.adoc:46]: @QuangHoang1210 @vttranlina you could put a label on your mail soon :D

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**